source/cloud/windows.md

```diff
@@ -33,33 +38,68 @@ Click or select the `Overview` tab, you will see the Windows admin password
+It is possible to customize and upload your own windows image. 
+The guide below is a semi-automated method of doing so. 
+## Preperation
+1) Download the latest stable virtio driver:
+https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/
+3) Save as `virtio-win.iso`
+4) Download the latest windows iso, save as `Windows.iso`
+* [Win10](https://www.microsoft.com/en-us/software-download/windows10ISO)
+* [Win11](https://www.microsoft.com/software-download/windows11)
+5) Configure/Download the unattended install iso: [https://schneegans.de/windows/unattend-generator/](https://schneegans.de/windows/unattend-generator/?LanguageMode=Unattended&UILanguage=en-US&UserLocale=nl-BE&KeyboardLayout=0409%3A00000409&GeoLocation=21&ProcessorArchitecture=amd64&BypassRequirementsCheck=true&ComputerNameMode=Custom&ComputerName=openstack&TimeZoneMode=Implicit&PartitionMode=Interactive&WindowsEditionMode=Unattended&WindowsEdition=pro&UserAccountMode=Unattended&AccountName0=Admin&AccountPassword0=&AccountGroup0=Administrators&AutoLogonMode=None&PasswordExpirationMode=Unlimited&LockoutMode=Default&EnableRemoteDesktop=true&DisableAppSuggestions=true&DisableWidgets=true&PreventDeviceEncryption=true&WifiMode=Skip&ExpressSettings=DisableAll&SystemScript0=Set-Content+-Path+C%3A%5CWindows%5Ccloudconfig.ps1+-Value+%22Add-Content+-Path+%27C%3A%5CProgram+Files%5CCloudbase+Solutions%5CCloudbase-Init%5Cconf%5Ccloudbase-init*%27+-Value+%27first_logon_behaviour%3Dno%27%22&SystemScriptType0=Ps1&FirstLogonScript0=pnputil+-i+-a+E%3A%5Cviostor%5Cwin10%5Camd64%5C*.INF&FirstLogonScriptType0=Cmd&FirstLogonScript1=Set-ExecutionPolicy+Unrestricted&FirstLogonScriptType1=Ps1&FirstLogonScript2=Remove-Partition+-Disknumber+0+-PartitionNumber+%24%28Get-Partition+%7C+Select-Object+-Last+1%29.PartitionNumber+-Confirm%3A%24false&FirstLogonScriptType2=Ps1&FirstLogonScript3=Invoke-WebRequest+-UseBasicParsing+https%3A%2F%2Fgithub.com%2Fcloudbase%2Fcloudbase-init%2Freleases%2Fdownload%2F1.1.4%2FCloudbaseInitSetup_1_1_4_x64.msi+-OutFile+cloudbaseinit.msi%3B+cloudbaseinit.msi+&FirstLogonScriptType3=Ps1&WdacMode=Skip)(link includes current configuration)
+
+## Creating the image
+Create the disk: 
+```
+qemu-img create -f qcow2 windows.qcow2 40G
+```
+Install windows on the disk, mounting all the required drives:
+```
+virt-install --connect qemu:///system   --name windows --ram 6144 --vcpus 2   --network network=default,model=virtio --disk path=windows.qcow2,format=qcow2,device=disk,bus=virtio --cdrom Windows.iso --disk path=unattend.iso,device=cdrom --disk path=virtio-win.iso,device=cdrom   --vnc --os-variant=win10
+```
+Or `--os-variant=win11` for win11.
+
+### Windows setup
+Most windows setup options are taken care of by `unattend.iso` except the partitioning. 
+#### Drivers
+In the partitioning window, click `load drivers` and browse to `NetKVM\w10\amd64` on the virtio drive and select that folder. (Obviously select `w11` if on Windows 11), then `next`.
```

Comment:
should specify they should browse this within virtio-win.iso cdrom, add a screenshot to make it more clear to the user